okhttp-dnsoverhttps/src/main/java/okhttp3/dnsoverhttps/DnsOverHttps.kt

```diff
@@ -57,55 +57,8 @@ class DnsOverHttps internal constructor(
   @get:JvmName("includeIPv6") val includeIPv6: Boolean,
   @get:JvmName("post") val post: Boolean,
   @get:JvmName("resolvePrivateAddresses") val resolvePrivateAddresses: Boolean,
-  @get:JvmName("resolvePublicAddresses") val resolvePublicAddresses: Boolean,
-  private val systemDns: Dns,
-  private val bootstrapDnsHosts: List<InetAddress>?
+  @get:JvmName("resolvePublicAddresses") val resolvePublicAddresses: Boolean
 ) : Dns {
-
-  @JvmName("-deprecated_url")
-  @Deprecated(
-      message = "moved to val",
-      replaceWith = ReplaceWith(expression = "url"),
-      level = DeprecationLevel.WARNING)
-  fun url(): HttpUrl = url
-
-  @JvmName("-deprecated_post")
-  @Deprecated(
-      message = "moved to val",
-      replaceWith = ReplaceWith(expression = "post"),
-      level = DeprecationLevel.WARNING)
-  fun post(): Boolean = post
-
-  @JvmName("-deprecated_includeIPv6")
-  @Deprecated(
-      message = "moved to val",
-      replaceWith = ReplaceWith(expression = "includeIPv6"),
-      level = DeprecationLevel.WARNING)
-  fun includeIPv6(): Boolean = includeIPv6
-
-  @JvmName("-deprecated_client")
-  @Deprecated(
-      message = "moved to val",
-      replaceWith = ReplaceWith(expression = "client"),
-      level = DeprecationLevel.WARNING)
-  fun client(): OkHttpClient = client.newBuilder()
```

Comment:
this was broken; the actual client val is what gets used, but only this one that is configured properly.

Reply:
Are we going to write API compatibility doc for 4.0.0-ALPHA02 or 4.0.0-RC1 in CHANGELOG.md? 👀